### ISSUE TITLE
Page accessibility

### ISSUE THREAD

**Issue by fcalderan** [cmpolis/Pagify#4]
I tried to disable javascript and reload demo page and no text appeared. ti could be a better choice to make the page work also without js (with every section already displayed in page in case of no js support)

**Comment by crazy4groovy** [comment]
I don't think this is possible, since JavaScript would be needed to fetch and display the pages. An alternative (as I did) would be to have default text in the final content area that asks the user to turn on JavaScript in their browser.
